recipe/meta.yaml

```diff
@@ -44,18 +45,19 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp >=8,<9
+    - arrow-cpp >=10,<11
     - pyarrow
     - numpy
```

Comment:
Not sure numpy is required.

Reply:
I'm guessing numpy is probably automatically pulled by pyarrow though...

Reply:
make sense